docs/source/vissl_modules/models.rst

```diff
@@ -58,13 +57,12 @@ Examples of trunks:
         MODEL:
           TRUNK:
             NAME: regnet
-            TRUNK_PARAMS:
-              REGNET:
-                depth: 27
-                w_0: 640
-                w_a: 230.83
-                w_m: 2.53
-                group_width: 373
+            REGNET:
+            depth: 27
+            w_0: 640
+            w_a: 230.83
+            w_m: 2.53
+            group_width: 373
```

Comment:
this needs to be nested under `REGNET`

Reply:
@prigoyal, that is a good catch. Thank you.